src/main/kotlin/app/revanced/patches/youtube/layout/comments/patch/CommentsPatch.kt

```diff
@@ -44,10 +56,48 @@ class CommentsPatch : ResourcePatch {
                         StringResource("revanced_hide_preview_comment_on", "Preview comment is hidden"),
                         StringResource("revanced_hide_preview_comment_off", "Preview comment is shown")
                     ),
+                    SwitchPreference(
+                        "revanced_hide_shorts_comments_button",
+                        StringResource("revanced_hide_shorts_comments_button_title", "Hide shorts comments button"),
+                        false,
+                        StringResource("revanced_hide_shorts_comments_button_on", "Shorts comments button is hidden"),
+                        StringResource("revanced_hide_shorts_comments_button_off", "Shorts comments button is shown")
+                    ),
                 ),
                 StringResource("revanced_comments_summary", "Manage the visibility of comments section components")
             )
         )
+
+        val shortsCommentsButtonResult = ShortsCommentsButtonFingerprint.result!!
+        val shortsCommentsButtonMethod = shortsCommentsButtonResult.mutableMethod
+
+        val checkCastAnchorFingerprint = object : MethodFingerprint(
+            opcodes = listOf(
+                Opcode.CONST,
+                Opcode.CONST_HIGH16,
+                Opcode.IF_EQZ,
+                Opcode.CONST,
+                Opcode.INVOKE_STATIC,
+                Opcode.MOVE_RESULT_OBJECT,
+                Opcode.CHECK_CAST,
+            )
+        ) {}
+        val checkCastAnchorIndex = checkCastAnchorFingerprint.also {
+            it.resolve(context, shortsCommentsButtonMethod, shortsCommentsButtonResult.classDef)
+        }.result!!.scanResult.patternScanResult!!.endIndex
+
+        shortsCommentsButtonMethod.addInstructions(
+            checkCastAnchorIndex + 1, """
+                invoke-static {v${(shortsCommentsButtonMethod.instruction(checkCastAnchorIndex) as OneRegisterInstruction).registerA}}, Lapp/revanced/integrations/patches/HideShortsCommentsButtonPatch;->hideShortsCommentsButton(Landroid/view/View;)V
+            """
+        )
+
         return PatchResultSuccess()
     }
+
+    internal companion object {
+        internal var shortsCommentsButtonId: Long = ResourceMappingResourcePatch.resourceMappings.single {
```

Comment:
This is unsafe, an explanation is available in another patch with similar usage.

Reply:
Done! I took the sponsorblock fingerprint as an example.

Reply:
Done! I took the ThemeSetterSystem fingerprint as example.